source/dbo/Stored Procedures/Data Migration/uspMigrationDuplicateAlerts.sql

```diff
@@ -4,41 +4,51 @@ AS
 	SELECT
 		mrr.MigrationNotificationId													AS 'MigrationNotificationId',
-		original.ETSID																AS 'EtsId',
-		original.NotificationId														AS 'NtbsId',
-		original.NotificationDate													AS 'NotificationDate',
-		originalTbService.[Name]													AS 'TbService',
-		duplicate.NotificationId													AS 'DuplicateNtbsId',
-		duplicate.ETSID																AS 'DuplicateEtsId',
-		duplicate.NotificationDate													AS 'DuplicateNotificationDate',
-		duplicateTbService.[Name]													AS 'DuplicateTbService',
-		ABS(DATEDIFF(day, original.NotificationDate, duplicate.NotificationDate))	AS 'DaysBetweenNotifications',
+		firstNot.ETSID																AS 'EtsId',
+		firstNot.NotificationId														AS 'NtbsId',
+		firstNot.NotificationDate													AS 'NotificationDate',
+		firstNotTbService.[Name]													AS 'TbService',
+		secondNot.NotificationId													AS 'DuplicateNtbsId',
+		secondNot.ETSID																AS 'DuplicateEtsId',
+		secondNot.NotificationDate													AS 'DuplicateNotificationDate',
+		secondNotTbService.[Name]													AS 'DuplicateTbService',
+		ABS(DATEDIFF(day, firstNot.NotificationDate, secondNot.NotificationDate))	AS 'DaysBetweenNotifications',
 		CASE
-			WHEN originalp.GivenName = duplicatep.GivenName
-				AND originalp.FamilyName = duplicatep.FamilyName
-				AND originalp.NhsNumber = duplicatep.NhsNumber
-				AND originalp.Dob = duplicatep.Dob
+			WHEN firstNotP.GivenName = secondNotP.GivenName
+				AND firstNotP.FamilyName = secondNotP.FamilyName
+				AND firstNotP.NhsNumber = secondNotP.NhsNumber
+				AND firstNotP.Dob = secondNotP.Dob
 				THEN 'Yes'
 			ELSE 'No'
 		END																			AS 'DemographicsMatch'
 	FROM DuplicatePairs pairs
-		LEFT JOIN [dbo].[MigrationRunResults] mrr ON mrr.NTBSNotificationId = pairs.OriginalNotification
-		LEFT JOIN [$(NTBS)].[dbo].[Notification] original on original.NotificationId = pairs.OriginalNotification
-		LEFT JOIN [$(NTBS)].[dbo].[Patients] originalp on originalp.NotificationId = original.NotificationId
-		LEFT JOIN [$(NTBS)].[dbo].[HospitalDetails] originalHd on originalHd.NotificationId = original.NotificationId
-		LEFT JOIN [$(NTBS)].[ReferenceData].[TbService] originalTbService on originalTbService.Code = originalHd.TBServiceCode
-		LEFT JOIN [$(NTBS)].[dbo].[Notification] duplicate on duplicate.NotificationId = pairs.DuplicateNotification
-		LEFT JOIN [$(NTBS)].[dbo].[Patients] duplicatep on duplicatep.NotificationId = duplicate.NotificationId
-		LEFT JOIN [$(NTBS)].[dbo].[HospitalDetails] duplicateHd on duplicateHd.NotificationId = duplicate.NotificationId
-		LEFT JOIN [$(NTBS)].[ReferenceData].[TbService] duplicateTbService on duplicateTbService.Code = duplicateHd.TBServiceCode
\ No newline at end of file
+		LEFT JOIN [dbo].[MigrationRunResults] mrr ON mrr.NTBSNotificationId = pairs.FirstNotification
+		LEFT JOIN [$(NTBS)].[dbo].[Notification] firstNot on firstNot.NotificationId = pairs.FirstNotification
```

Comment:
Minor thing, but I don't really like `Not` as an abbreviation of `Notification` as it has a meaning of its own. I probably would have gone with `firstNotification` and then just `firstPatient` etc. That said, SQL table names are often less meaningful than this!